### PR TITLE
ABU-736 fix

### PR DESCRIPTION
**Pull request by zenduo**

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1 you having fun? *turns off phone notifications*

**Comment by zenduo** [comment]
Lol thats them all now!

**Comment by zenduo** [comment]
Following a discussion with @oliverfoster and @taylortom we suggest the following:

- Remove licence and author from all JS files
- Remove authors from bower
- Add link to issues page
- In the read me, link to the contributors page on github (will need to be in another pr) e.g. https://github.com/adaptlearning/adapt-contrib-confidenceSlider/graphs/contributors
- Additionally, if this receives 3+1's, you are happy that all plugins can be merged by one reviewer other than me.

If you could all review and +1 this if agreed, I will roll out across all plug-ins.

**Comment by oliverfoster** [comment]
agreed - don't count mine as a plus-one, i've already said my bit.

**Comment by dancgray** [comment]
+1

**Comment by taylortom** [comment]
+1 :dancer:

**Comment by lc-thomasberger** [comment]
+1

File: bower.json [hunk 1]
```diff
@@ -2,10 +2,7 @@
   "name": "adapt-contrib-confidenceSlider",
   "version": "0.0.1",
   "homepage": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider",
-  "authors": [
-    "Kev Adsett <kev.adsett@kineo.com>",
-    "Himanshu Rajotia <himanshu.rajotia@credipoint.com>"
-  ],
+  "issues": "https://github.com/adaptlearning/adapt-contrib-confidenceSlider/issues",
```
Comment (moloko) [inline bower.json]: this should be link to Jira really

**Comment by zenduo** [comment]
Regarding @moloko message above, all core items have been done, anything non-core points to the github repo issues.

**Comment by taylortom** [comment]
I think that's fair enough - we need to decide upon a strategy for non-core plugins

**Pull request merged**